src/account-info/AccountDetailsScreen.js

```diff
@@ -37,13 +43,20 @@ class AccountDetailsScreen extends PureComponent<Props> {
         _: user.full_name || ' ',
       },
     };
+    let deactivatedText = null;
+    if (user.is_active === false) {
+      deactivatedText = (
+        <Label style={styles.deactivatedText} text="(This user/bot has been deactivated)" />
```

Comment:
This string needs to go in `messages_en.json`.

(Also, let's stick with `(This user has been deactivated)`, to mirror the webapp.)